tools/dependencies.props

```diff
@@ -1,7 +1,7 @@
 <Project>
   <PropertyGroup>
     <CoreFxStableVersion>4.3.0</CoreFxStableVersion>
-    <RoslynVersion>2.6.0-beta3-62316-02</RoslynVersion>
+    <RoslynVersion>2.8.0-beta4</RoslynVersion>
```

Comment:
I can't find 2.8.0-beta4 version of Microsoft.NETCore.Compilers within the build tools myget feed:
https://dotnet.myget.org/feed/dotnet-buildtools/package/nuget/Microsoft.NETCore.Compilers

There is a 2.8.0-beta4 version of Microsoft.Net.Compilers only:
https://dotnet.myget.org/feed/dotnet-buildtools/package/nuget/Microsoft.NET.Compilers

It looks like we are pulling this package from nuget.org: 
https://www.nuget.org/packages/Microsoft.NETCore.Compilers/2.8.0-beta4